SampleApp/nuget.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="Local" value="../build/" />
```

Comment:
I think this will work in CI but we'll find out. If building SampleApp is going to be part of our gradle tasks and we want to be able to work with local packages then we can't depend on individual VS configurations.